python/bottle/containerWithTests/Tests/functional_tests/test_functional.py

```diff
@@ -24,12 +24,14 @@ def test_selenium(self):
 				title = self.driver.title
 				self.assertIn("Home Page - Python Bottle Application", title)
 				break
-			except AssertionError:
+			except AssertionError as e:
```

Comment:
do we need to handle assertion error separately. for other language you are not doing that.

Reply:
Done